src/seqcover.nim

```diff
@@ -70,17 +72,50 @@ proc report_main() =
   var sample_d4s = read_d4s_to_table(opts.samples)
   stderr.write_line &"[seqcover] read {sample_d4s.len} sample coverage files"
   var gpt: seq[GenePlotData]
-  for gene in get_genes(opts.genes.split(","), hg19=opts.hg19):
-    var u = gene.transcripts.union
-    echo &"{u.chr}\t{u.txstart - 500}\t{u.txend + 500}"
-    var pd = gene.plot_data(sample_d4s, backgrounds, extend=10, fai=fa, max_gap=50)
-    gpt.add(pd)
+  
+  # If exists a transcripts-file - use instead of get_genes proc
+  var query_genes = opts.genes.split(",")
```

Comment:
can you put this new section in a function `get_genes()` or something like that to avoid (further) inflating this section.

Reply:
I moved the new code to the get_genes() proc in the utils.nim. Is that okay?